api/envoy/config/filter/http/ext_authz/v2/ext_authz.proto

```diff
@@ -53,6 +53,15 @@ message ExtAuthz {
 
   // Enables filter to buffer the client request body and send it within the authorization request.
   BufferSettings with_request_body = 5;
+
+  // Clears route cache in order to allow the external authorization service to correctly affect
+  // routing decisions. Filter clears all cached routes when:
+  //
+  // 1. The field is set to *true*.
+  // 2. The status returned from the authorization service is a HTTP 200 or gRPC 0.
+  // 3. One or more *authorization response headers* are addressed to the upstream.
```

Comment:
Sorry what does this mean? Can you clarify?

Reply:
I rephrased it.